src/util/log.cpp

```diff
@@ -91,8 +95,6 @@ Log::Log(LogLevel level_, std::ostream &ostream) : level(level_), stream(ostream
     }
 }
 
-Log::Log(LogLevel level_) : Log(level_, buffer) {}
```

Comment:
It seems this is considered as UB: we are using field of class itself(`buffer`) when calling constructor of this class.